index.html

```diff
@@ -7,6 +7,6 @@
     <title>Document</title>
 </head>
 <body>
-    
+ <h1>Hello Microvers!</h1>   
```

Comment:
- [ ] As stated in the project requirement Kindly Make sure that the your A .css file that includes **one class** with styles for your header (make it in your favorite color) ,

![Capture](https://user-images.githubusercontent.com/93598189/218444684-65cd0968-78a3-4f79-816d-8d1c02289fdc.PNG)